### PR TITLE
Refactoring: Move tutorial-like content and references to cratedb-guide

### PR DESCRIPTION
**Pull request by amotl**
## About

This patch refactors all the tutorial-like content and references to the new cratedb-guide documentation section.

## Details

Because of followup cross-linking, this patch depends on the other patch to be integrated beforehand. Currently, it will fail on CI.

- https://github.com/crate/cratedb-guide/pull/29

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -22,5 +22,8 @@
 html_static_path = ['_static']
 
 
+intersphinx_mapping["guide"] = ('https://cratedb-guide--29.org.readthedocs.build/', None)
```
Comment (amotl) [inline docs/conf.py:25]: Of course, 7098972c1c52 needs to be removed again before merging. It is only here to demonstrate everything fits together well, by making CI succeed.

**Review by seut — COMMENTED** [review]
Isn't this complete repository obsolete after the migrations and can be archived completely? If so, I'd suggest to not remove stuff with PR's but archive the complete repository afterwards.

**Comment by amotl** [comment]
The repository isn't obsolete. It carries all the content visible at https://cratedb.com/docs/crate/clients/. We can think about further refactoring on subsequent iterations.

**Review by seut — APPROVED** [review]

**Review by hammerhead — APPROVED** [review]

File: docs/conf.py (hunk 1 above)
Comment (amotl) [reply docs/conf.py]: The CrateDB Guide has been published, and this workaround has been removed.

**Pull request merged**